README.md

```diff
@@ -240,24 +240,20 @@ func main() {
 	dataCh := consumer.StartFetch()
 	fetchRes := make([]hstream.RecordId, 0, 100)
 	for res := range dataCh {
-		receivedRecords, err := res.GetResult()
-		if err != nil {
-			log.Printf("Fetch error: %s\n", err)
+		if res.Err != nil {
+			log.Printf("Fetch error: %s\n", res.Err)
 			continue
 		}
 
-		for _, record := range receivedRecords {
+		for _, record := range res.Result {
 			rid := record.GetRecordId()
 			log.Printf("receive recordId: %s\n", rid.String())
 			fetchRes = append(fetchRes, rid)
+      record.Ack()
```

Comment:
Indentation error, need to align code manually

Reply:
fixed, my bad vscode auto save setting